### PR TITLE
Typography optimisation: Heading styling 

### PR DESCRIPTION
**Pull request by myabc**
This PR contains low-risk, easy-to-implement changes to headings. This PR should not be confused with  with #2910 (User Story #[18330](https://community.openproject.org/work_packages/18330)), which is a much more comprehensive refactoring.

https://community.openproject.org/work_packages/19571

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://myabc-feature-quick-win-heading-styling.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://myabc-feature-quick-win-heading-styling.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/assets/stylesheets/open_project_global/_variables.sass [hunk 1]
```diff
@@ -34,7 +34,7 @@ $light-gray: #ccc
 
 $body-background:         white !default
 $body-font-family:        'Lato', 'Lucida Grande', Helvetica, Arial, sans-serif !default
-$body-font-color:         #555 !default
+$body-font-color:         #333 !default
```
Comment (NobodysNightmare) [inline app/assets/stylesheets/open_project_global/_variables.sass:37]: You can now safely remove the color rule from:

``` CSS
.inplace-edit--errors {
    line-height: 1.2em;
    color: #444;
}
```

The color was only added there to provide correct accessibility on red background.

Comment (NobodysNightmare) [inline app/assets/stylesheets/open_project_global/_variables.sass:37]: I like the new color btw. It looks more crispy!

Comment (myabc) [inline app/assets/stylesheets/open_project_global/_variables.sass:37]: @NobodysNightmare `#444` on a red background?

Comment (NobodysNightmare) [inline app/assets/stylesheets/open_project_global/_variables.sass:37]: It was for the error messages: http://leaverou.github.io/contrast-ratio/#%23444-on-%23fedada

The old `#555` did not get a 7:1 contrast, that is why we went for `#444`.

Now that your default is `#333` the extra rule is not needed anymore...

**Comment by NobodysNightmare** [comment]
Waiting for travis to concur and hope that you will remove the CSS rule I mentioned.

Otherwise LGTM :+1:

**Comment by NobodysNightmare** [comment]
Haha, thanks @myabc for pro-actively pushing that change to `release/4.1` xD

Merging this PR now ^^ Travis will surely agree...

**Pull request merged**